src/main/webapp/index.xhtml

```diff
@@ -1,49 +1,55 @@
-						href="public/track.xhtml">Public Tracking Interface</a>
-				</h2>
-				<h2>
-					<a class="shadowme" style="color: orange; text-decoration: none;"
-						href="admin/dashboard.xhtml">Administration Interface</a>
-				</h2>
-				<h2>
-					<a class="shadowme" style="color: orange; text-decoration: none;"
-						href="javascript:void(0)"
-						onclick="window.open('event-logger/index.xhtml', '_blank', 'width=320,height=440,popup=yes,menubar=no,toolbar=no,location=no,status=no,resizable=yes,scrollbars=yes,titlebar=yes,statusbar=no')">Mobile
-						Event Logger</a>
-				</h2>
-				<hr class="intro-divider" />
-			</div>
-		</div>
-	</div>
-</h:body>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://xmlns.jcp.org/jsf/html">
+
+<head>
+    <title>Eclipse Cargo Tracker</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <h:outputStylesheet library="css" name="title.css" />
+
+    <link rel="stylesheet"
+          href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" />
+    <link href="http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic,700italic"
+          rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+    <div class="intro-header">
+        <div class="row">
+            <div class="intro-message">
+                <hr class="intro-divider" />
+                <h1 class="shadowme">Eclipse Cargo Tracker</h1>
+                <a href="https://github.com/eclipse-ee4j/cargotracker/">
+                    <h:graphicImage library="images"
+                                     name="CTlogobadge128.png"
+                                     style="height: 64px; width: 64px;"
+                                     alt="Cargo Tracker logo badge"/>
+                </a>
+                <h2 class="shadowme">Applied domain-driven design blueprints for Jakarta EE</h2>
+                <h2>
+                    <a class="shadowme" style="color: orange; text-decoration: none;"
+                       href="public/track.xhtml">Public Tracking Interface</a>
+                </h2>
+                <h2>
+                    <a class="shadowme" style="color: orange; text-decoration: none;"
+                       href="admin/dashboard.xhtml">Administration Interface</a>
+                </h2>
+                <h2>
+                    <a class="shadowme" style="color: orange; text-decoration: none;"
+                       href="javascript:void(0)"
+                       onclick="window.open('event-logger/index.xhtml', '_blank', 'width=320,height=440,popup=yes,menubar=no,toolbar=no,location=no,status=no,resizable=yes,scrollbars=yes,titlebar=yes,statusbar=no')">Mobile Event Logger</a>
+                </h2>
+                <hr class="intro-divider" />
+            </div>
+        </div>
+    </div>
+</body>
+
 </html>
+

```

Comment:
Why is this added?

Reply:
The first code is a JSF page that uses tags from the JSF HTML namespace, such as <h:head>, <h:body>, <h:outputStylesheet>, and <h:graphicImage>. JSF is a framework for building web applications in Java that provides a component-based model for user interface development.

The second code is a standard HTML page that does not use any JSF tags. It includes the same CSS styles and images as the first code, but it uses standard HTML tags such as <head>, <body>, <link>, and <a>.

Both codes display the same content and have the same functionality, but the first code is a JSF page while the second code is a standard HTML page.

Reply:
I am doing undo of this change in the next PR after solving all the issues as you told sir.

Thanks and Regards

Reply:
Great, I will look for it. Kindly keep PRs focused on clear improvements.

Reply:
Ok Sure Sir i will do my best and keep on improving as per your guidance.

Thanks and Regards